src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="Sitelogo"
```

Comment:
The `alt` attribute of the `img` tag should describe the image content. The current value "Sitelogo" is too vague. Try to describe what the image is about. For example, if it's a logo of your company, you can write the company's name and what kind of company it is.